pages/gallery.html

```diff
@@ -1,12 +1,92 @@
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Guy's Gallery</title>
+  <link rel="stylesheet" href="../styles/style.css"> 
+  <link rel="stylesheet" href="../styles/gallery.css"> 
 </head>
 <body>
+  <header>
+    <h1>Guy's Gourmet Gallery</h1>
+    <nav>
+      <ul>
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="facts.html">Facts</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="container">
+
+    <div class="gallery-post">
```

Comment:
`div`s don't provide meaningful information about the structure of our pages so we should avoid using those elements unless it is purely for styling or no semantic element makes sense. I would recommend using something like `<article>` to replace these.